libs/lib-server/core/src/lib/extensions/resource-importer.ts

```diff
@@ -45,3 +47,77 @@ export interface ImportsRefAgent {
    */
   getPackageRef(contribType: ContributionType, aliasRef: string): string;
 }
+
+/**
+ * Helper to get the required action details for a handler associated with actionId
+ * @example
+ * App.json
+ *  ```{
+ *    ...,
+ *    triggers: [
+ *      handlers: [
+ *        id: 'some-action-id'
+ *      ]
+ *    ],
+ *    actions: [
+ *      {
+ *        id: 'some-action-id',
+ *        ref: 'some-plugin-resource-ref'
+ *        settings: {
+ *          resourceMapping: 'resource-id',
+ *          someOtherProperty: 'value'
+ *        }
+ *      }
+ *    ],
+ *  }```
+ */
+export interface ImportsActionsManager {
```

Comment:
since this doesn't manage anything maybe it should be called registry? `ImportActionsRegistry`? Or perhaps ÌmportActionsDirectory`?

Reply:
Renamed it to `ImportActionsRegistry`